### PR TITLE
`DEX` - FAQ screen

### LINKED ISSUE #1852: `DEX` - FAQ
#### What would you like to be added:
We would like to add some FAQ content for DEX.

```
# Decentralized Exchange

The decentralized exchange function will allow users of DeFiChain to swap cryptocurrencies in a peer-to-peer fashion. The decentralized exchange function matches people for trading directly, without the need to buy and sell currency through an exchange. 

## Transaction fees

Each transaction will be subject to a small amount of fees. The amount may vary depending on the network's congestion.

## What is slippage tolerance?

Due to the dynamic nature of the blockchain, prices on the DEX change every block. This results in **slippage**, which is the difference between the final transaction price and the price displayed on the app before transaction confirmation. The slippage tolerance feature allows you to indicate the maximum slippage you are willing to accept in a transaction. If the slippage for your transaction is higher than your slippage tolerance, the transaction will not be completed and the original amount will be returned to your wallet.
```
#### Why is this needed:
To give more context on our knowledge base.

- [ ] Add on Knowledge base (Settings -> About -> Knowledge Base) as another entry (DEX)
- [ ] Translations

### PR DESCRIPTION
**Pull request by chloezxyy**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:

Fixes #1852

#### Which issue(s) does this PR fixes?:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [x] Unit tests*
- [x] Added e2e tests*
- [x] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/WalletNavigator/screens/CreateWallet/DexFaq.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      title: translate('components/DexFaq', 'Transaction fees'),
+      content: [{
+        text: translate('components/DexFaq', 'Each transaction will be subject to a small amount of fees. The amount may vary depending on the network\'s congestion.'),
+        type: 'paragraph'
+      }]
+    },
+    {
+      title: translate('components/DexFaq', 'What is slippage tolerance?'),
+      content: [{
+        text: translate('components/DexFaq', 'Due to the dynamic nature of the blockchain, prices on the DEX change every block. \n\nThis results in **slippage**, which is the difference between the final transaction price and the price displayed on the app before transaction confirmation. \n\nThe slippage tolerance feature allows you to indicate the maximum slippage you are willing to accept in a transaction. \n\nIf the slippage for your transaction is higher than your slippage tolerance, the transaction will not be completed and the original amount will be returned to your wallet.'),
```
Comment (thedoublejay) [inline mobile-app/app/screens/WalletNavigator/screens/CreateWallet/DexFaq.tsx:18]: I guess the added asterisks on `**slippage**` is intended to make it bolder. Will not work, let's just remove it

**Review by thedoublejay — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/AppNavigator/screens/Settings/SettingsNavigator.tsx [hunk 2]
```diff
@@ -239,6 +254,7 @@ export function SettingsNavigator (): JSX.Element {
           headerBackTitleVisible: false
         }}
       />
+
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Settings/SettingsNavigator.tsx:257]: can we remove this line?

File: shared/translations/languages/zh-Hant.json [hunk 3]
```diff
@@ -12,7 +12,8 @@
     "Refresh": "刷新",
     "Refresh recovery words": "刷新恢復字組",
     "You are about to generate a new set of recovery words. Continue?": "您正要產生新的恢復字組，確認要繼續？",
-    "Passcode FAQ": "密碼常見問題"
+    "Passcode FAQ": "密碼常見問題",
+    "DEX FAQ": "DEX"
```
Comment (kyleleow) [inline shared/translations/languages/zh-Hant.json:16]: pending translation?

File: shared/translations/languages/zh-Hans.json [hunk 4]
```diff
@@ -12,7 +12,8 @@
     "Refresh": "刷新",
     "Refresh recovery words": "刷新恢复字组",
     "You are about to generate a new set of recovery words. Continue?": "您正要产生新的恢复字组，确认要继续？",
-    "Passcode FAQ": "密码常见问题"
+    "Passcode FAQ": "密码常见问题",
+    "DEX FAQ": "DEX"
```
Comment (kyleleow) [inline shared/translations/languages/zh-Hans.json:16]: pending translation?

File: mobile-app/app/screens/WalletNavigator/screens/CreateWallet/DexFaq.test.tsx [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import { render } from '@testing-library/react-native'
+
```
Comment (kyleleow) [inline mobile-app/app/screens/WalletNavigator/screens/CreateWallet/DexFaq.test.tsx:2]: can we remove this line?

**Review by kyleleow — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/WalletNavigator/screens/CreateWallet/DexFaq.tsx (hunk 1 above)
Comment (chloezxyy) [reply mobile-app/app/screens/WalletNavigator/screens/CreateWallet/DexFaq.tsx:18]: noted

File: mobile-app/app/screens/AppNavigator/screens/Settings/SettingsNavigator.tsx (hunk 2 above)
Comment (chloezxyy) [reply mobile-app/app/screens/AppNavigator/screens/Settings/SettingsNavigator.tsx:257]: noted

File: mobile-app/app/screens/WalletNavigator/screens/CreateWallet/DexFaq.test.tsx (hunk 5 above)
Comment (chloezxyy) [reply mobile-app/app/screens/WalletNavigator/screens/CreateWallet/DexFaq.test.tsx:2]: noted

File: shared/translations/languages/zh-Hant.json (hunk 3 above)
Comment (chloezxyy) [reply shared/translations/languages/zh-Hant.json]: rectified

File: shared/translations/languages/zh-Hans.json (hunk 4 above)
Comment (chloezxyy) [reply shared/translations/languages/zh-Hans.json]: rectified

File: mobile-app/app/screens/WalletNavigator/screens/CreateWallet/LiquidityMiningFaq.test.tsx [hunk 6]
```diff
@@ -0,0 +1,11 @@
+import { render } from '@testing-library/react-native'
+
```
Comment (kyleleow) [inline mobile-app/app/screens/WalletNavigator/screens/CreateWallet/LiquidityMiningFaq.test.tsx:2]: can we remove this line

File: mobile-app/app/screens/AppNavigator/screens/Settings/SettingsNavigator.tsx [hunk 7]
```diff
@@ -226,6 +228,33 @@ export function SettingsNavigator (): JSX.Element {
...
+          headerTitle: () => (
+            <HeaderTitle
+              text={translate('screens/WalletNavigator', 'DEX FAQ')}
+              containerTestID={headerContainerTestId}
+            />
+          ),
+          headerBackTitleVisible: false
+        }}
+      />
+
```
Comment (kyleleow) [inline mobile-app/app/screens/AppNavigator/screens/Settings/SettingsNavigator.tsx:244]: can remove this line too

File: shared/translations/languages/de.json [hunk 8]
```diff
@@ -12,7 +12,9 @@
     "Refresh": "Erneuern",
     "Refresh recovery words": "Erneuere Wiederherstellungssatz",
     "You are about to generate a new set of recovery words. Continue?": "Du generierst einen neuen Satz Wiederherstellungswörter. Möchtest du fortfahren?",
-    "Passcode FAQ": "Passwort FAQ"
+    "Passcode FAQ": "Passwort FAQ",
+    "DEX FAQ": "DEX FAQ",
+    "Liquidity Mining FAQ":"Liquiditäts-Mining FAQ"
```
Comment (kyleleow) [inline shared/translations/languages/de.json:17]:
```suggestion
    "Liquidity Mining FAQ": "Liquiditäts-Mining FAQ" 
```

File: shared/translations/languages/de.json [hunk 9]
```diff
@@ -218,7 +220,8 @@
     "Passcode": "Passwort",
     "UTXO vs Token": "UTXO vs Token",
     "Loans": "Darlehen",
-    "Auctions": "Auktionen"
+    "Auctions": "Auktionen",
+    "Liquidity Mining":"Liquiditäts-Mining"
```
Comment (kyleleow) [inline shared/translations/languages/de.json:224]:
```suggestion
    "Liquidity Mining": "Liquiditäts-Mining" 
```

File: shared/translations/languages/de.json [hunk 10]
```diff
@@ -665,6 +668,30 @@
...
+    "FREQUENTLY ASKED QUESTIONS": "HÄUFIG GESTELLTE FRAGEN"
+  },
+  "components/LiquidityMiningFaq" : {
+    "Liquidity Mining":"Liquiditäts-Mining",
```
Comment (kyleleow) [inline shared/translations/languages/de.json:681]:
```suggestion
    "Liquidity Mining": "Liquiditäts-Mining",
```
can you help to double check the rest of the formatting in your code changes for this `.json`, i believe there are some auto-formatting shortcut for your IDE

File: shared/translations/languages/zh-Hant.json [hunk 11]
```diff
@@ -12,7 +12,9 @@
     "Refresh": "刷新",
     "Refresh recovery words": "刷新恢復字組",
     "You are about to generate a new set of recovery words. Continue?": "您正要產生新的恢復字組，確認要繼續？",
-    "Passcode FAQ": "密碼常見問題"
+    "Passcode FAQ": "密碼常見問題",
+    "DEX FAQ": "DEX 問題",
```
Comment (kyleleow) [inline shared/translations/languages/zh-Hant.json:16]:
```suggestion
    "DEX FAQ": "DEX 常見問題",
```

File: shared/translations/languages/zh-Hant.json [hunk 12]
```diff
@@ -12,7 +12,9 @@
...
+    "Liquidity Mining FAQ":"流動性挖礦问题"
```
Comment (kyleleow) [inline shared/translations/languages/zh-Hant.json:17]:
```suggestion
    "Liquidity Mining FAQ":"流動性挖礦常見問題"
```

File: shared/translations/languages/zh-Hant.json [hunk 13]
```diff
@@ -670,6 +673,29 @@
...
+    "Transaction fees": "交易費用",
+    "Each transaction will be subject to a small amount of fees. The amount may vary depending on the network's congestion.": "每筆交易需要少量的交易費用。交易費用會根據網絡的流量情況產生變化。",
+    "What is slippage tolerance?": "什麼是交易滑價?",
+    "Due to the dynamic nature of the blockchain, prices on the DEX change every block. \n\nThis results in slippage, which is the difference between the final transaction price and the price displayed on the app before transaction confirmation. \n\nThe slippage tolerance feature allows you to indicate the maximum slippage you are willing to accept in a transaction. \n\nIf the slippage for your transaction is higher than your slippage tolerance, the transaction will not be completed and the original amount will be returned to your wallet.": "由於區塊鏈的動態性質，DEX上的價格隨著每個區塊而變。 \n\n這導致了 **滑價**，也就是確認交易前顯示出的價格和最終交易價格之間的差異。 \n\n程序中出現的「交易滑價」可自訂您可接受的滑價範圍。\n\n假設交易滑點高於您選擇的交易滑價百分比，那麼交易不會完成，交易金額則退回到您的錢包餘額。",
+    "FREQUENTLY ASKED QUESTIONS": "常見問題"
+  },
+  "components/LiquidityMiningFaq" : {
+    "Liquidity Mining":"流動性挖礦",
```
Comment (kyleleow) [inline shared/translations/languages/zh-Hant.json:686]: same thing for the spacing format

**Review by kyleleow — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/AppNavigator/screens/Settings/SettingsNavigator.tsx (hunk 2 above)
Comment (kyleleow) [reply mobile-app/app/screens/AppNavigator/screens/Settings/SettingsNavigator.tsx:270]: missed this?

File: shared/translations/languages/de.json [hunk 14]
```diff
@@ -665,6 +668,30 @@
...
+    "Due to the dynamic nature of the blockchain, prices on the DEX change every block. \n\nThis results in slippage, which is the difference between the final transaction price and the price displayed on the app before transaction confirmation. \n\nThe slippage tolerance feature allows you to indicate the maximum slippage you are willing to accept in a transaction. \n\nIf the slippage for your transaction is higher than your slippage tolerance, the transaction will not be completed and the original amount will be returned to your wallet.": "Aufgrund der dynamischen Natur der Blockchain ändern sich die Preise auf der DEX mit jedem Block. \n\nDaraus resultiert Slippage, also die Differenz zwischen dem endgültigen Transaktionspreis und dem Preis, der vor der Transaktionsbestätigung in der App angezeigt wird. \n\nMit der Slippage-Toleranzfunktion kannst du die maximale Slippage angeben, die du bei einer Transaktion zu akzeptieren bereit bist. \n\nWenn die Abweichung für deine Transaktion höher ist als deine Slippage-Toleranz, wird die Transaktion nicht abgeschlossen und der ursprüngliche Betrag wird auf deine Wallet zurücktransferiert.",
```
Comment (thedoublejay) [inline shared/translations/languages/de.json:677]: There might be an issue on this translation

**Review by thedoublejay — CHANGES_REQUESTED** [review]

**Review by kyleleow — APPROVED** [review]

**Review by thedoublejay — DISMISSED** [review]

**Review by thedoublejay — APPROVED** [review]

**Review by kyleleow — APPROVED** [review]

**Review by fullstackninja864 — APPROVED** [review]

**Pull request merged**